go.mod

```diff
@@ -34,9 +34,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.18.0-rc.1
 	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
-	sigs.k8s.io/controller-runtime v0.3.1-0.20191016212439-2df793d02076
-	sigs.k8s.io/controller-tools v0.2.9-0.20200331153640-3c5446d407dd
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/controller-tools v0.3.0
```

Comment:
Will exclude those changes, when PR https://github.com/openshift/machine-api-operator/pull/570 will be merged, as right now it is complicating testing in my environment.